ecology.yaml

```diff
@@ -0,0 +1,515 @@
+---
+install_repository_dependencies: true
+install_resolver_dependencies: true
+install_tool_dependencies: false
+
+tools:
+
+  - name: 'collapse_collections'
+    owner: 'nml'
+    tool_panel_section_label: 'Collection Operations'
+
+  - name: dwt_var_perfeature
```

Comment:
I condafied this one: https://github.com/galaxyproject/tools-devteam/pull/510

Reply:
Thank you!